app/models.py

```diff
@@ -130,6 +130,7 @@ class QuestionResponse(Question):
     """Model for the response of any request that returns a question"""
 
     question_set_id: str
+    text: Optional[str] = None
```

Comment:
why is this added? `text` is already present in `Question` model and we are inheriting it for `QuestionResponse` model